### ISSUE TITLE
[QUESTION] MIL for Audio data

### ISSUE THREAD

**Issue by JBJoyce** [Franblueee/torchmil#22]
Is it possible to use torchmil for weakly supervised learning of audio data using modern, pretrained audio models (e.g., Whisper, WavLM)?

I have audio data which is labeled at an utterance level, but would like to train/fine-tune a model to predict labels at a subutterance level. Most of the examples presented here are based around computer vision tasks so I wanted to ensure this would be possible.

Thank you!
